src/main.js

```diff
@@ -59,6 +60,18 @@ let load = base.loader({
     },
   },
 
+  AmiSet: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let AmiSet = AmiSet.setup({
```

Comment:
slight bug here.  You are declaring a new variable, `AmiSet`, but assigning the first value to something that uses the variable declared here.  This alone will cause
```
localhost:~/taskcluster/aws-provisioner $ node
> let x = x;
ReferenceError: x is not defined
    at repl:1:31
    at REPLServer.defaultEval (repl.js:252:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:417:12)
    at emitOne (events.js:82:20)
    at REPLServer.emit (events.js:169:7)
    at REPLServer.Interface._onLine (readline.js:210:10)
    at REPLServer.Interface._line (readline.js:549:8)
    at REPLServer.Interface._ttyWrite (readline.js:826:14)
```
What you're looking for here is
```javascript
      let AmiSet = amiSet.setup({ 
```
because you imported the `AmiSet` 'class' using the name `amiSet` above, where you `require()` the file.